### PR TITLE
Adds craftable and wearable flower crowns

### PR DESCRIPTION
**Pull request by SynthTwo**
## What Does This PR Do

Ports wearable and craftable flower crowns from https://github.com/BeeStation/BeeStation-Hornet/pull/5397. 

Geranium flower crown is new however, and a recolour of the poppy crown. :3c

## Why It's Good For The Game

Can't really do much with flowers in game currently, so this is a cute option for players. Also, more interaction with Botany and the garden as a whole.

## Images of changes

I have fixed the way it looks on Grey, I am just too lazy to show it. Ignore the vox puke.

image

## Testing

Loaded the server, made flower crowns in the crafting menu, and wore them on my head. No skrells were harmed during this. 

## Changelog
:cl:
add: Wearable and craftable flower crowns. Found in the crafting menu under Clothing.
/:cl:

### CODE REVIEW

**Comment by 1080pCat** [comment]
image
wow

**Comment by DGamerL** [comment]
> image wow

It's true

File: code/modules/crafting/tailoring.dm [hunk 1]
```diff
@@ -280,3 +280,39 @@
 				/obj/item/assembly/voice = 1,
 				/obj/item/stack/cable_coil = 5)
 	category = CAT_CLOTHING
+
+/datum/crafting_recipe/flower_crown
+	name = "Flower Crown"
+	result = list(/obj/item/clothing/head/flower_crown)
+	reqs = list(/obj/item/food/snacks/grown/poppy = 3, /obj/item/food/snacks/grown/poppy/lily = 3,
+	/obj/item/grown/sunflower = 3,)
```
Comment (DGamerL) [inline code/modules/crafting/tailoring.dm:288]:
```suggestion
	reqs = list(/obj/item/food/snacks/grown/poppy = 3, 
					/obj/item/food/snacks/grown/poppy/lily = 3,  
					/obj/item/grown/sunflower = 3
					)
```

File: code/modules/crafting/tailoring.dm [hunk 2]
```diff
@@ -280,3 +280,39 @@
...
+	time = 20
```
Comment (DGamerL) [inline code/modules/crafting/tailoring.dm:289]:
```suggestion
	time = 2 SECONDS
```

File: code/modules/crafting/tailoring.dm [hunk 3]
```diff
@@ -280,3 +280,39 @@
...
+	category = CAT_CLOTHING
+
+/datum/crafting_recipe/sunflower_crown
+	name = "Sunflower Crown"
+	result = list(/obj/item/clothing/head/sunflower_crown)
+	reqs = list(/obj/item/grown/sunflower = 5)
+	time = 20
```
Comment (DGamerL) [inline code/modules/crafting/tailoring.dm:296]:
```suggestion
	time = 2 SECONDS
```

File: code/modules/crafting/tailoring.dm [hunk 4]
```diff
@@ -280,3 +280,39 @@
...
+	category = CAT_CLOTHING
+
+/datum/crafting_recipe/poppy_crown
+	name = "Poppy Crown"
+	result = list(/obj/item/clothing/head/poppy_crown)
+	reqs = list(/obj/item/food/snacks/grown/poppy = 5)
+	time = 20
```
Comment (DGamerL) [inline code/modules/crafting/tailoring.dm:303]:
```suggestion
	time = 2 SECONDS
```

File: code/modules/crafting/tailoring.dm [hunk 5]
```diff
@@ -280,3 +280,39 @@
...
+	result = list(/obj/item/clothing/head/lily_crown)
+	reqs = list(/obj/item/food/snacks/grown/poppy/lily = 5)
+	time = 20
+	category = CAT_CLOTHING
+
+/datum/crafting_recipe/geranium_crown
+	name = "Geranium Crown"
+	result = list(/obj/item/clothing/head/geranium_crown)
+	reqs = list(/obj/item/food/snacks/grown/poppy/geranium = 5)
+	time = 20
```
Comment (DGamerL) [inline code/modules/crafting/tailoring.dm:317]:
```suggestion
	time = 2 SECONDS
```

**Review by DGamerL — COMMENTED** [review]
🌻🌺🌸🌻🌺🌸🌻🌺🌸🌻🌺🌸

**Comment by SynthTwo** [comment]
FUCKED UP.

**Pull request closed without merging**